### PR TITLE
[DT KIT] Versioning updated + minor changes

### PR DESCRIPTION
**Pull request by BirgitBoss**
Table on Versioning: more text added
+
typos, formatting, adding text for clarity

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by arnoweiss — APPROVED** [review]
Changes only clarify existing documentation. No objections.

File: docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md
Comment (florianrusch-zf) [inline docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md:25]:
```suggestion
The Asset Administration Shell (AAS) is a specification that is released by the [Industrial Digital Twin Association (IDTA)](https://industrialdigitaltwin.org/)
with a perspective to be adopted by the [International Electrotechnical Commission (IEC)](https://www.iec.ch/homepage).
```

**Review by florianrusch-zf — APPROVED** [review]

**Review by SebastianBezold — APPROVED** [review]
Cannot say much about the contents, but trusting @arnoweiss and @florianrusch-zf  on that 😁 .
Nothing else I can see, that would prohibit a merge! 🚀

**Comment by florianrusch-zf** [comment]
@BirgitBoss do you want to fix the spaces, or should we hit the merge button?

**Comment by maximilianong** [comment]
Any updates here? I think that can be merged - please confirm @BirgitBoss

**Comment by arnoweiss** [comment]
This is superseeded by #654 - I branched from Birgit's work and continued there since I figured that this wouldn't get merged. If is is now merged, there might be conflicts as we worked on the same documents. If Birgit agrees, I'd recommend closing this PR and focussing on #654 instead.

**Comment by maximilianong** [comment]
@danielmiehle and me did some clean up work on the open PRs yesterday. Let's wait another 24h for the feedback of @BirgitBoss . If not, I would close it.

**Comment by BirgitBoss** [comment]
@florianrusch-zf thanks for your review, I just accepted the changed (or sorry: I did an own commit but same changes)

**Comment by florianrusch-zf** [comment]
LGTM

@arnoweiss @maximilianong if you created changes based on the commits from @BirgitBoss you should be able to simply do a git rebase after this PR got merged.

**Comment by arnoweiss** [comment]
This is gonna require rebasing #654 as it goes to the same document, so I'm obviously not stoked about this. Won't block the merge though.

**Review by arnoweiss — APPROVED** [review]

**Review by danielmiehle — APPROVED** [review]

**Pull request merged**
